src/AutoStateTransitions/Repos/WorkItemRepo.cs

```diff
@@ -29,43 +29,39 @@ public WorkItemRepo(IOptions<AppSettings> appSettings, IHelper helper)
 
         public async Task<WorkItem> GetWorkItem(VssConnection connection, int id)
         {
-            using (WorkItemTrackingHttpClient client = connection.GetClient<WorkItemTrackingHttpClient>())
-            {
-
-                try
-                {
-                    return await client.GetWorkItemAsync(id, null, null, WorkItemExpand.Relations);
+            WorkItemTrackingHttpClient client = connection.GetClient<WorkItemTrackingHttpClient>();
 
-                }
-                catch (Exception)
-                {
-                    return null;
-                }
+            try
+            {
+                return await client.GetWorkItemAsync(id, null, null, WorkItemExpand.Relations);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
```

Comment:
Don't log exception directly to Console. Maybe MS Extensions Logging should be used?